app/assets/javascripts/application.js

```diff
@@ -16,3 +16,13 @@
 //= require_tree .
 
 $(function(){ $(document).foundation(); });
+
+def reverse_string(string)
+  new_string = []
+  i = string.length-1
+  while i >= 0
+    new_string.push(string[i])
+    i -= 1
+  end
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>'end' is not defined.